src/fragments/lib/logging/common/remote_configuration/remote_configuration.mdx

```diff
@@ -0,0 +1,230 @@
+    const configFileName = 'remoteloggingconstraints.json'
+    const logGroupName = <CloudWatch-log-group-name>
+    const authRoleName = <Amplify-authenticated-role-name> 
+    const unAuthRoleName = <Amplify-unauthenticated-role-name>
+    const bucketName = <S3-bucket-name>
+
+    new logs.LogGroup(this, 'Log Group', {
+      logGroupName: logGroupName,
+      retention: logs.RetentionDays.INFINITE
+    })
+    
+    const authRole = iam.Role.fromRoleName(this, "Auth-Role", authRoleName)
+    const unAuthRole = iam.Role.fromRoleName(this, "UnAuth-Role", unAuthRoleName)
+    const logResource = `arn:aws:logs:${region}:${account}:log-group:${logGroupName}:log-stream:*`
+    const logIAMPolicy = new iam.PolicyStatement({
+      effect: iam.Effect.ALLOW,
+      resources: [logResource],
+      actions: ["logs:PutLogEvents", "logs:DescribeLogStreams", "logs:CreateLogStream"]
+    })
+
+    authRole.addToPrincipalPolicy(logIAMPolicy)
+    unAuthRole.addToPrincipalPolicy(logIAMPolicy)
+
+    const remoteConfigBucket = new s3.Bucket(this, 'AmplifyRemoteLogging-Bucket', {
+        publicReadAccess: false,
+        versioned: true,
+        bucketName: bucketName
+      });
+
+    new BucketDeployment(this, `AmplifyRemoteLogging-BucketDeployment`, {
+        sources: [
+            Source.asset(path.dirname(path.join(loggingConfigLocation))),
+        ],
+        destinationBucket: remoteConfigBucket
+    });
+
+    const handler = new lambda.Function(this, "AmplifyRemoteLogging-Handler", {
+      runtime: lambda.Runtime.NODEJS_18_X,
+      code: lambda.Code.fromAsset(path.dirname(path.join(lambdaConfig))),
+      handler: "remotelogging.main",
+      environment: {
+        BUCKET: remoteConfigBucket.bucketName,
+        KEY: configFileName
+      }
+    })
+
+    remoteConfigBucket.grantRead(handler)
+
+    const api = new apigateway.RestApi(this, "AmplifyRemoteLogging-API", {
+      restApiName: "Logging API",
+      description: "API Gateway for Remote Logging"
+    })
+
+    const getRemoteLoggingIntegration = new apigateway.LambdaIntegration(handler)
+    const loggingConstraints = api.root.addResource('loggingconstraints')
+    const getLoggingConstraints = loggingConstraints.addMethod('GET', getRemoteLoggingIntegration, {
+      authorizationType: apigateway.AuthorizationType.IAM
+    })
+
+    const apiInvokePolicy = new iam.PolicyStatement({
+      effect: iam.Effect.ALLOW,
+      resources: [ getLoggingConstraints.methodArn ],
+      actions: ['execute-api:Invoke']
+    })
+
+    authRole.addToPrincipalPolicy(apiInvokePolicy)
+    unAuthRole.addToPrincipalPolicy(apiInvokePolicy)
+
+    new cdk.CfnOutput(this, 'LoggingConstraintsURL', {
+      value: `https://${api.restApiId}.execute-api.${region}.amazonaws.com/prod/loggingconstraints`,
+    });
+    new cdk.CfnOutput(this, 'CloudWatchLogGroupName', { value: logGroupName });
+    new cdk.CfnOutput(this, 'CloudWatchRegion', { value: region });
```

Comment:
We will also need to cfnoutput the endpoint - and make it clear at the end of the code snippet that the developer will need to add that endpoint to the config

Reply:
This is already defined, renamed to `APIEndpoint` to avoid confusion and added text after the code snippet to say that values will be printed in terminal.